go/api/response_handlers.go

```diff
@@ -0,0 +1,55 @@
+// Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0
+
+package api
+
+// #cgo LDFLAGS: -L../target/release -lglide_rs
+// #include "../lib.h"
+import "C"
+
+import (
+	"fmt"
+	"go/types"
+	"reflect"
+	"unsafe"
+)
+
+type redisResponse interface {
+	interface{} | string | types.Nil
```

Comment:
are these the only types, or these are the types required for `SET` and `GET` which you implemented?